gaptrain/calculators.py

```diff
@@ -391,6 +391,52 @@ def run_cp2k(configuration, max_force):
     return configuration
 
 
+def get_gp_var_quip_out(configuration, out_filename='quip.out'):
+    """
+    Given a QUIP output file extract the list of atomic variances for each
+    set of atoms
+
+    :param configuration:
```

Comment:
Not sure if this is the right version I'm looking at (#18) but the docstrings are still missing

Reply:
I thought I got them all 😞